torch/distributed/fsdp/_fsdp.py

```diff
@@ -0,0 +1,69 @@
+from typing import Callable, cast, Iterable, Optional, Union
```

Comment:
nit: assume you are going to move it to '_composable' folder?

Reply:
Yup! I will do that in follow-ups.